dbservice/postgresdb/insert.go

```diff
@@ -0,0 +1,44 @@
+package postgresdb
+
+import (
+	"fmt"
+
+	"github.com/jmoiron/sqlx"
+)
+
+var insert = func(db *sqlx.DB, table, id, columnName2, value2, columnName3, value3 string) error {
+	var i int
+	if err := db.Get(&i, fmt.Sprintf("SELECT COUNT(*) FROM %s WHERE (%s=$1 AND %s=$2)", table, "id", columnName2), id, value2); err != nil {
```

Comment:
why you are using constant `"id"` as parameter to `fmt.Sprintf()`?

Reply:
removed it